packages/stryker-html-reporter/package.json

```diff
@@ -41,21 +41,19 @@
     "mkdirp": "~0.5.1",
     "mz": "~2.7.0",
     "rimraf": "~2.6.1",
+    "stryker-api": "~0.23.0",
```

Comment:
Why pin it on exactly `0.23.0`? (`~` pins on the minor version which is the last `0` here.

Reply:
Because it now is a _dependency_ instead of a peerDependency. It will automatically be updated by lerna (at least, as far as I understand it).